integrations/gong/syncs/users.ts

```diff
@@ -0,0 +1,27 @@
+import type { NangoSync, ProxyConfiguration, User } from '../../models.js';
+import type { GongUser } from '../types.js';
+
+export default async function fetchData(nango: NangoSync) {
+    const config: ProxyConfiguration = {
+        // https://gong.app.gong.io/settings/api/documentation#post-/v2/users/extensive
+        endpoint: `/v2/users/extensive`,
+        paginate: {
+            type: 'cursor',
+            cursor_name_in_request: 'cursor',
+            cursor_path_in_response: 'cursor',
+            response_path: 'users'
+        },
+        method: 'post',
+        retries: 10
+    };
+
```

Comment:
should use `createdFromDateTime` to make the sync incremental instead of full and then no need to track deletes

Reply:
this has been updated